MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
- [ ] [OPTIONAL] Well done for adding an MIT license just as requested 👏. Please consider editing the license further to replace the [YOUR NAME] placeholder with your real name. Also try to replace all the other placeholders with the actual values you want in the license 👍

Reply:
That is great, Thanks @codepantha, I'll do the changes 🥳🥳